### PR TITLE
Added | AnimeTW

### PR DESCRIPTION
**Pull request by Zerlingg**
Browsing

Playing

Paused

### CODE REVIEW

File: websites/A/AnimeTW/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  "author": {
+    "name": "Giingu",
+    "id": "156481332652802048"
+  },
+  "service": "AnimeTW",
+  "description": {
+    "en": "AnimeTW is a streaming website where you can watch anime with Spanish subtitles.",
+    "zh_CN": "AnimeTW是一个视频网站，您可以在其中观看带有西班牙语字幕的动漫。"
+  },
+  "url": ["animetw.net"],
```
Comment (Bas950) [inline websites/A/AnimeTW/dist/metadata.json:11]: url doesnt need to be in an array

File: websites/A/AnimeTW/dist/metadata.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  "version": "1.0",
+  "logo": "https://i.ibb.co/2nk4xJS/Sin-t-tulo.jpg",
+  "thumbnail": "https://i.ibb.co/TqmncHy/image.png",
```
Comment (Bas950) [inline websites/A/AnimeTW/dist/metadata.json:14]: upload to imgur

File: websites/A/AnimeTW/dist/metadata.json [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+    "id": "156481332652802048"
+  },
+  "service": "AnimeTW",
+  "description": {
+    "en": "AnimeTW is a streaming website where you can watch anime with Spanish subtitles.",
+    "zh_CN": "AnimeTW是一个视频网站，您可以在其中观看带有西班牙语字幕的动漫。"
+  },
+  "url": ["animetw.net"],
+  "version": "1.0",
+  "logo": "https://i.ibb.co/2nk4xJS/Sin-t-tulo.jpg",
```
Comment (Bas950) [inline websites/A/AnimeTW/dist/metadata.json:13]: version 1.0.0

File: websites/A/AnimeTW/dist/metadata.json [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+    "id": "156481332652802048"
+  },
+  "service": "AnimeTW",
+  "description": {
+    "en": "AnimeTW is a streaming website where you can watch anime with Spanish subtitles.",
+    "zh_CN": "AnimeTW是一个视频网站，您可以在其中观看带有西班牙语字幕的动漫。"
+  },
+  "url": "animetw.net",
+  "version": "1.0.0",
+  "logo": "https://i.imgur.com/NHoHFWf.jpg",
```
Comment (frozaum) [inline websites/A/AnimeTW/dist/metadata.json:13]: Hello, can you change the logo with one that has better quality or prove that the logo is supposed to be like this resolution?

Comment (Zerlingg) [reply websites/A/AnimeTW/dist/metadata.json:13]: they do not have any logo with better quality, they didnt even have a background and i had to add it

Comment (Zerlingg) [reply websites/A/AnimeTW/dist/metadata.json:13]: https://animetw.net/ check the site yourself and try finding their logo on google images, theres nothing else

**Comment by Zerlingg** [comment]
Okay i made a better logo myself

**Pull request merged**